package.json

```diff
@@ -0,0 +1,29 @@
+{
+    "name": "lwc-mobile-samples",
+    "private": true,
+    "version": "0.0.1",
+    "description": "LWC Mobile Samples",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/forcedotcom/LWC-Mobile-Samples"
+    },
+    "license": "MIT",
```

Comment:
Our license is CC0 1.0 Universal. We should change this to something like CC0 or CC0-1.0.

Reply:
oh good catch